rust/src/chain_crypto/evolving.rs

```diff
@@ -0,0 +1,29 @@
+use crate::chain_crypto::key::{AsymmetricKey, SecretKey};
```

Comment:
Removed KES implementation.

Reply:
As an FYI this is the answer I got from IOHK on the KES implementation

> We are not using the Rust KES impl. We've had the one we are using audited, but we did want to compare to the rust one as a reference.
We spent some time trying to compare the two impls but got to a bit of an impasse. There are two Rust impls, the "naive recursive" one and the "production" one. There are no Haskell bindings for the recursive one so it makes it hard to compare. It appears however that the recursive one at least would produce the same size output as we expect (and as expected from the paper). Then the "production" one must be implementing a different algorithm since it produces a different sized signature, so it is not comparable.